JOSESwift/Sources/Algorithms.swift

```diff
@@ -96,6 +96,13 @@ public enum HMACAlgorithm: String {
     }
 }
 
+/// An algorithm for Thumbprint calculation.
+///
+/// - SHA256
+public enum ThumbprintAlgorithm: String {
```

Comment:
```suggestion
public enum JWKThumbprintAlgorithm: String {
```

To be able to more clearly distinguish this from `HMACAlgorithm` it might make sense to add the `JWK` prefix.